MC/run/ANCHOR/anchorMC.sh

```diff
@@ -0,0 +1,192 @@
+baseargs="-tf ${NTIMEFRAMES} --split-id ${SPLITID} --prod-split ${PRODSPLIT} --cycle ${CYCLE} --run-number ${RUNNUMBER}"
+
+# these arguments will be passed as well but only evetually be digested by o2dpg_sim_workflow.py which is called from o2dpg_sim_workflow_anchored.py
+remainingargs="-gen pythia8 -proc heavy_ion -seed ${SEED} -ns ${NSIGEVENTS} --include-local-qc --pregenCollContext"
+remainingargs="${remainingargs} -e ${SIMENGINE} -j ${NWORKERS}"
+remainingargs="${remainingargs} -productionTag ${ALIEN_JDL_LPMPRODUCTIONTAG:-alibi_anchorTest_tmp}"
+remainingargs="${remainingargs} --anchor-config config-json.json"
+
+echo "baseargs: ${baseargs}"
+echo "remainingargs: ${remainingargs}"
+
+# query CCDB has changed, w/o "_"
+${O2DPG_ROOT}/MC/bin/o2dpg_sim_workflow_anchored.py ${baseargs} -- ${remainingargs} &> timestampsampling_${RUNNUMBER}.log
+if [ "$?" != "0" ] ; then
+    echo "Problem during anchor timestamp sampling and workflow creation. Exiting."
+    exit 1
+fi
+
+TIMESTAMP=`grep "Determined timestamp to be" timestampsampling_${RUNNUMBER}.log | awk '//{print $6}'`
+echo "TIMESTAMP IS ${TIMESTAMP}"
+
+# -- PREFETCH CCDB OBJECTS TO DISC      --
+# (make sure the right objects at the right timestamp are fetched
+#  until https://alice.its.cern.ch/jira/browse/O2-2852 is fixed)
+# NOTE: In fact, looking at the ticket, it should be fixed now. However, not changing at the moment as further tests would be needed to confirm here.
+
+CCDBOBJECTS="/CTP/Calib/OrbitReset /GLO/Config/GRPMagField/ /GLO/Config/GRPLHCIF /ITS/Calib/DeadMap /ITS/Calib/NoiseMap /ITS/Calib/ClusterDictionary /TPC/Calib/PadGainFull /TPC/Calib/TopologyGain /TPC/Calib/TimeGain /TPC/Calib/PadGainResidual /TPC/Config/FEEPad /TOF/Calib/Diagnostic /TOF/Calib/LHCphase /TOF/Calib/FEELIGHT /TOF/Calib/ChannelCalib /MFT/Calib/DeadMap /MFT/Calib/NoiseMap /MFT/Calib/ClusterDictionary /FV0/Calibration/ChannelTimeOffset"
+
+${O2_ROOT}/bin/o2-ccdb-downloadccdbfile --host http://alice-ccdb.cern.ch/ -p ${CCDBOBJECTS} -d .ccdb --timestamp ${TIMESTAMP}
+if [ ! "$?" == "0" ]; then
+  echo "Problem during CCDB prefetching of ${CCDBOBJECTS}. Exiting."
+  exit 1
+fi
+
+# -- Create aligned geometry using ITS and MFT ideal alignments to avoid overlaps in geant
+CCDBOBJECTS_IDEAL_MC="ITS/Calib/Align MFT/Calib/Align"
+TIMESTAMP_IDEAL_MC=1
+${O2_ROOT}/bin/o2-ccdb-downloadccdbfile --host http://alice-ccdb.cern.ch/ -p ${CCDBOBJECTS_IDEAL_MC} -d .ccdb --timestamp ${TIMESTAMP_IDEAL_MC}
+if [ ! "$?" == "0" ]; then
+  echo "Problem during CCDB prefetching of ${CCDBOBJECTS_IDEAL_MC}. Exiting."
+  exit 1
+fi
+
+echo "run with echo in pipe" | ${O2_ROOT}/bin/o2-create-aligned-geometry-workflow --configKeyValues "HBFUtils.startTime=${TIMESTAMP}" --condition-remap=file://${ALICEO2_CCDB_LOCALCACHE}=ITS/Calib/Align,MFT/Calib/Align -b
+mkdir -p $ALICEO2_CCDB_LOCALCACHE/GLO/Config/GeometryAligned
+ln -s -f $PWD/o2sim_geometry-aligned.root $ALICEO2_CCDB_LOCALCACHE/GLO/Config/GeometryAligned/snapshot.root
```

Comment:
I defined `ALICEO2_CCDB_LOCALCACHE` above explicitly now.
The reasons why it might have worked before
1. `ALICEO2_CCDB_LOCALCACHE` was always set in the parent stop of this script
1. It actually failed but the workflow could still run, downloading everything when it was needed
1. It actually failed but the workflow could still run, files are not actually needed